src/disco/pages/DiscoPane/index.js

```diff
@@ -167,7 +167,8 @@ export class DiscoPaneBase extends React.Component<InternalProps> {
         {this.renderFindMoreButton({ position: 'top' })}
 
         {results.map((item) => (
-          <Addon addon={item} key={item.guid} />
+          // $FLOW_FIXME: addon should be a AddonType
```

Comment:
could you provide a type hint like `results.map((item: AddonType) => ...)`. Otherwise, you lose a lot of coverage on this line.

Reply:
are you sure given that `results` is typed?

Reply:
Ah I see what you mean I guess, well not now but soon-ish everything will have a correct type. Right now, it is being weird. I'll try your idea though.

Reply:
That creates a huge list of Flow errors :/

Reply:
In any case, this was (and still is) fixed in #6325.